applications/nrf_desktop/doc/dfu.rst

```diff
@@ -47,19 +47,65 @@ If the buffer is small, the host must perform the DFU progress synchronization m
 MCUboot bootloader mode
 =======================
 
-The MCUboot bootloader can either move the image to the primary slot before booting it (``swap mode``) or boot the image directly from the secondary slot (``direct-xip mode``).
+This module supports the following modes of the MCUboot bootloader:
 
-If the MCUboot bootloader in the swap mode is selected, the DFU module does the following:
+* ``swap`` (:kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_SWAP_USING_MOVE`)
+* ``direct-xip`` (:kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_DIRECT_XIP`)
+* ``RAM load`` (:kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_RAM_LOAD`)
+
+.. note::
+   Other bootloader modes may be compatible with this module, but they are not actively tested as part of the nRF Desktop configurations.
+
+See the :ref:`nrf_desktop_bootloader` section for more information on the MCUboot bootloader configuration.
+
+Swap mode
+---------
+
+MCUboot in the swap mode moves the image to the primary slot before booting it.
+
+If this bootloader mode is selected, the DFU module does the following:
 
  * Requests the image upgrade after the whole image is transferred over the :ref:`nrf_desktop_config_channel`.
  * Confirms the running image after the device is rebooted.
 
-If the MCUboot bootloader's direct-xip mode is used, the module does not mark the newly uploaded image as pending and does not confirm it after a successful boot.
+.. note::
+   The MCUboot bootloader supports a few variants of the swap mode that are differentiated by the algorithm used to move the image to the primary slot.
+   The nRF Desktop configurations use the default swap mode variant indicated by the :kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_SWAP_USING_MOVE` Kconfig option.
+   Other variants, indicated by the :kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_SWAP_USING_OFFSET` or :kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_SWAP_WITHOUT_SCRATCH` Kconfig options, are not actively tested in the nRF Desktop application.
+
+The DFU module does not rely on any Kconfig option that indicates the MCUboot bootloader swap mode, as the default module behavior is compatible with this mode.
+In the swap mode, the DFU module reports the ``MCUBOOT`` bootloader name.
```

Comment:
Consider keeping the reported bootloader names as part of the list under `This module supports the following modes of the MCUboot bootloader;`.